packages/relayer/src/verify.rs

```diff
@@ -89,8 +95,15 @@ pub fn verify_signature(
     let message_hash: [u8; 32] = Sha256::digest(message).into();
     let signature_bytes = signature.as_slice();
     let pubkey_bytes = pubkey.as_slice();
+    // If pubkey is compressed (33 bytes), uncompress it
+    let pubkey_bytes = if pubkey_bytes.len() == 33 {
```

Comment:
The comment is good and doesn't make a `const` necessary, but I think it would be nice to have, just in case that value is reused elsewhere